core/dbt/config/profile.py

```diff
@@ -20,7 +20,7 @@
 DEFAULT_THREADS = 1
 DEFAULT_PROFILES_DIR = os.path.join(os.path.expanduser('~'), '.dbt')
 PROFILES_DIR = os.path.expanduser(
-    os.environ.get('DBT_PROFILES_DIR', DEFAULT_PROFILES_DIR)
+    os.getenv('DBT_PROFILES_DIR', DEFAULT_PROFILES_DIR)
```

Comment:
I changed all `os.environ.get` -> `os.getenv` for grep-ability reason when I was adding some secret magic threading-related flags to the RPC server.